src/smclarify/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.3"
+__version__ = "0.4"
```

Comment:
Do we actually need to bump up the lib version?

Reply:
version bump up is not for the python 3.10 support. we made some api changes for bias_stats metrics and model report.